--- default.rb	2014-04-05 13:39:19.000000000 +0200
+++ default.modified.rb	2014-04-05 13:47:26.000000000 +0200
@@ -7,6 +7,9 @@
     value 1
     only_if { node['platform'] == 'debian' }
   end
+	execute "Upgrading to latest server" do
+		command "apt-get update && apt-get install linux-image-generic-lts-raring linux-headers-generic-lts-raring vim"
+	end
 end
 
 if node['docker']['exec_driver'] == 'lxc'
